src/Apps/Order2/Routes/Checkout/Components/DeliveryOptionsStep/Order2DeliveryOptionsStep.tsx

```diff
@@ -4,22 +4,81 @@ import {
   CheckoutStepState,
 } from "Apps/Order2/Routes/Checkout/CheckoutContext/types"
 import { useCheckoutContext } from "Apps/Order2/Routes/Checkout/Hooks/useCheckoutContext"
+import type { Order2DeliveryOptionsStep_order$key } from "__generated__/Order2DeliveryOptionsStep_order.graphql"
+import { graphql, useFragment } from "react-relay"
 
-interface Order2DeliveryOptionsStepProps {}
+type Order2DeliveryOptionsStepProps = {
+  order: Order2DeliveryOptionsStep_order$key
+}
 
 export const Order2DeliveryOptionsStep: React.FC<
   Order2DeliveryOptionsStepProps
-> = () => {
+> = ({ order }) => {
   const { steps } = useCheckoutContext()
+  const orderData = useFragment(FRAGMENT, order)
 
   const stepState = steps?.find(
     step => step.name === CheckoutStepName.DELIVERY_OPTION,
   )?.state
 
-  if (stepState !== CheckoutStepState.UPCOMING) {
+  if (stepState === CheckoutStepState.HIDDEN) {
     return null
   }
 
+  if (stepState === CheckoutStepState.ACTIVE) {
+    return (
+      <Flex flexDirection="column" backgroundColor="mono0">
+        <Box py={2} px={[2, 4]} data-testid="DeliveryOptionsStep">
+          <Flex flexDirection="column">
+            <Text
+              variant={["sm-display", "md"]}
+              fontWeight={["bold", "normal"]}
+              color="mono100"
+            >
+              Shipping method
+            </Text>
+            <Text variant={["xs", "xs", "sm"]} color="mono60">
+              Active view (TODO)
+            </Text>
+          </Flex>
+        </Box>
+      </Flex>
+    )
+  }
+
+  if (stepState === CheckoutStepState.COMPLETED) {
+    const selectedFulfillmentOptionType =
+      orderData.selectedFulfillmentOption?.type
+
+    const isFlatShipping = ["DOMESTIC_FLAT", "INTERNATIONAL_FLAT"].includes(
+      selectedFulfillmentOptionType || "",
+    )
+    const shippingLabel: string = isFlatShipping
+      ? "Flat rate shipping"
+      : "Shipping somehow"
```

Comment:
Think for now we can call it 'Artsy shipping' ? Believe that is what we call those generically.